apps/communications/contacts/js/views/selector.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+/* global fb, Contacts, contacts, utils */
+
+Contacts.Selector = (function(document, window, contacts, fb, utils){
```

Comment:
please do not use here the ```fb``` object as this functionality has nothing to do with Facebook